src/js/store/factory/marks/Area.ts

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+import {Record, RecordOf} from 'immutable';
+import {AreaMark, OnEvent} from 'vega-typings';
+
+const anchorTarget = require('../../../util/anchor-target');
+const test = require('../../../util/test-if');
+const propSg = require('../../../util/prop-signal');
+
+export type LyraAreaMark = {
```

Comment:
I wonder if it's worth using a more general `LyraMark` if the `_id` is the only field that's ever being added? I wonder if you can do something like `LyraMark<AreaMark>`?

Reply:
filed an issue https://github.com/vega/lyra/issues/436